src/site/content/en/blog/image-policies/index.md

```diff
@@ -0,0 +1,265 @@
+images in WebP lossy format if you can. If that is not sufficient, try WebP
+lossless format. Use JPEG on browsers that don't support WebP formats. Use PNG
+if none of thes formats work.
+
+If you are using WebP formats, try with stricter thresholds:
+
+* 0.2 for WEBPV8
+* 0.5 for WEBPL
+
+#### More examples
+
+```
+Feature-Policy:  unoptimized-lossy-images *(0.5);
+                 unoptimized-lossless-images *(1.0);
+                 unoptimized-lossless-images-strict *(1.0);
+```
+
+This policy is enforced on all origins with a threshold value of 0.5 (for lossy
+formats) and 1 (for lossless formats). Any `<img>` element whose image has a
+byte-per-pixel ratio exceeding the constraint is disallowed and will be replaced
+with a placeholder image.
+
+```
+Feature-Policy: unoptimized-lossy-images *(inf) 'self'(0.3);
+                unoptimized-lossless-images *(inf) 'self'(0.8);
+                unoptimized-lossless-images-strict *(inf) 'self'(0.8);
+```
+
+This policy is enforced on the site's origin with a threshold value of 0.3 (for
+lossy formats) and 0.8 (for lossless formats). The `<img>` elements in top-level
+browsing contexts and same origin nested browsing contexts will only render
+normally if the byte-per-pixel ratio meets these constraints. The `<img>` elements
+everywhere else will render normally.
+
+### Report-only mode in the wild
+
+Publishing a site with placeholder images may not be what you want. You can use
+the policies in enforcement mode (with unoptimized images rendered as
+placeholder images) during development and staging, and use report-only mode in
+production. (Check out [Feature Policy
+Reporting](https://github.com/w3c/webappsec-feature-policy/blob/master/reporting.md)
+for more details.) Similar to `Feature-Policy` HTTP header, the
+`Feature-Policy-Report-Only` header lets you observe violation reports in the
+wild without any enforcement.
+
+### Limitations
+
+Image policies only work on HTML image elements (`<img>`, `<source>`,
+etc.) and are not yet supported on background images or generated content. If you
+would like to have policies supported on broader contents, please let us know.
+
+## Optimizing your images
+
+I've talked quite a bit about optimizing your images, but haven't said how to do it. That topic is out of scope for this article, but you can learn more from the links below and from the codelabs listed at the end of the article.
+
+* [Optimizing images](https://web.dev/fast#optimize-your-images)
+* [Serve images with correct dimentions](https://web.dev/serve-images-with-correct-dimensions/)
+
+## Experiment with the policies in origin trials
+
+Image policies are available in Chrome 75 via an origin trial.
+
+To participate
+
+1. [request a token](https://developers.chrome.com/origintrials/#/view_trial/2562548187973812225)
+
+1. Add the token on any pages in your origin using an `Origin-Trial` HTTP
+  header:<br/>
+  <br/>
+  `Origin-Trial: **token as provided in the developer console**`
```

Comment:
This part seems not to render correctly in the viewer. Could you please verify? It is supposed to show code.

Reply:
Here's what it will look like rendered.
![image](https://user-images.githubusercontent.com/8204171/58210091-3e649680-7c9d-11e9-9388-853f614bfec2.png)

Reply:
ohhh my bad. Maybe mine is missing some nodejs stuff. Sorry about that.